src/test/java/uk/gov/digital/ho/hocs/casework/api/CaseDocumentServiceTest.java

```diff
@@ -0,0 +1,93 @@
+package uk.gov.digital.ho.hocs.casework.api;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.digital.ho.hocs.casework.client.documentclient.DocumentClient;
+import uk.gov.digital.ho.hocs.casework.client.documentclient.DocumentDto;
+import uk.gov.digital.ho.hocs.casework.client.documentclient.GetDocumentsResponse;
+import uk.gov.digital.ho.hocs.casework.client.documentclient.S3Document;
+
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CaseDocumentServiceTest {
+
+    private final UUID caseUUID = UUID.randomUUID();
+    private final UUID documentUUID = UUID.randomUUID();
+    private final String caseType = "MIN";
+
+    @Mock
+    private DocumentClient documentClient;
+    private CaseDocumentService caseDocumentService;
+
+    @Before
+    public void setUp() {
+        caseDocumentService = new CaseDocumentService(documentClient);
+    }
+
+    @Test
+    public void getDocuments() {
+        GetDocumentsResponse documentsResponse = mock(GetDocumentsResponse.class);
```

Comment:
It is best to instantiate actual instances of GetDocumentResponse, DocumentDto and S3Document in these tests, so that the values can be verified after the call to the service.